src/node_api.cc

```diff
@@ -940,7 +942,9 @@ const char* error_messages[] = {nullptr,
                                 "The async work item was cancelled",
                                 "napi_escape_handle already called on scope",
                                 "Invalid handle scope usage",
-                                "Invalid callback scope usage"};
+                                "Invalid callback scope usage",
+                                "Queue is full",
+                                "Handle is closing"};
```

Comment:
I think a bit more context in the message would be useful.  For example 'Threadsafe function queue is full'